docs/installation/conf_templates/ha_evcc_template.yaml

```diff
@@ -0,0 +1,317 @@
+## Configuration template with demo entities (meters, chargers) ##
```

Comment:
Wir können auf keinen Fall eine weitere Kopie der `evcc.dist.yaml` hier reinpacken. Es ist jetzt schon schwer, die Doku in Sync zu halten. Was ich sehen würde wäre ein art `starter yaml` wo wirklich nur die Basics (1x grid, 1x pv, 1x battery, 1x charger, 1x ladepunkt, 1x fahrzeug und evtl. ein fester Tarif) drin sind. Alles andere würd ich da rausnehmen und dann auf die [Referenze Docs](https://docs.evcc.io/docs/reference/configuration), [evcc.dist.yaml ](https://github.com/evcc-io/evcc/blob/master/evcc.dist.yaml) und `evcc configure` verweisen.

Reply:
Die "Starter Yaml" könnte sogar ein eigener Menüpunkt unter Installation werden. Diese Herrausforderung haben ja aktuell alle Installationswegen. Nicht nur Home Assistant.

Reply:
Meinst du die Starter yaml auch im code repo zu hinterlegen? 
Amrk. Meine demo yaml ist im Grunde keine Kopie von evcc.dist.yaml, sie enthält nur demo entities. 
Ich denke auch es macht  Sinn die Konfiguration in einem separaten Kapitel zu machen, nach der manuellen Installation am Schluss?

Reply:
> Meinst du die Starter yaml auch im code repo zu hinterlegen?

In meiner Wunschvorstellung wäre die so kurz und minimal, dass wir die auch Inline anzeigen könnten.

> Meine demo yaml ist im Grunde keine Kopie von evcc.dist.yaml, sie enthält nur demo entities.

Mir geht es hier primär um die ganzen anderen Optionen und Hilfstexte, die da jetzt mit drinstehen (und später aktualisiert werden müssen). Die hast du ja nicht neu geschrieben.

> Ich denke auch es macht Sinn die Konfiguration in einem separaten Kapitel zu machen, nach der manuellen Installation am Schluss?

Ja, ich würde es wirklich als eigene Unterseite von "Installation" sehen. Da könnten wir dann die unterschiedlichen Wege auflisten, die man hat, um an eine Config zu kommen: 1) `evcc configure`, 2) selbst von Grund auf aufbauen (Starter YAML) oder 3) mit der evcc.dist.yaml anfangen.

Auf dieser Seite können wir dann auch alle  relevanten Quellen verlinken, wo man dann mehr Details zu den Optionen herausfinden kann. Also "Referenz > evcc.yaml" und die evcc.dist.yaml (gh link). Auch Teile der FAQ (bspw. https://docs.evcc.io/docs/faq#fehlersuche) könnten dann mit auf diese Seite verschoben werden.

Noch ein Gedanke zur `evcc.dist.yaml` im Hauptrepo. Wir könnten überlegen, ob wir die so verändern, dass sie ein guter Startpunkt wird. Also die Dummy-Komponenten fix dort schon einbauen und alles Optionale wirklich erstmal auszukommentieren. Dann hätten wir nicht das Problem der Dopplung.

Reply:
Ja ich denke auch das wäre das beste wie du es vorschlägst. 

> Noch ein Gedanke zur evcc.dist.yaml im Hauptrepo. Wir könnten überlegen, ob wir die so verändern, dass sie ein guter Startpunkt wird. Also die Dummy-Komponenten fix dort schon einbauen und alles Optionale wirklich erstmal auszukommentieren. Dann hätten wir nicht das Problem der Dopplung.

Hätte den Vorteil, dass man nicht überall den links darauf hinterher rennen muss. Man könnte sich dann ev. überlegen eine Art von blueprints config yaml zu machen (z.b. Kopie von bestehender evcc.dist.yaml ). Für verschiedene Anwendungs Szenarien. Aber dann müssten die dann auch wieder gepflegt werden, oder klar als unsupported zu bezeichnen. 

Wann denkt ihr denn dass eine Umstellung auf UI config möglich sein könnte? Spielt hier auch mit rein zur Entscheidungsfindung. Wenn das schrittweise geschieht, müssten dann die config templates immer angepasst werden, d.h. die nicht mehr unterstützte Parameter gelöscht werden. 

Ev. könnte eine Tutorial Sektion helfen diesen Transformations Prozess möglichst reibungslos zu gestalten. Nach meinen Erfahrungen mit der Inbetriebname meines neuen OCPP Chargers und des gesamten Systems habe ich festgestellt, dass man als neuer user zu sehr auf die (sehr hilfbereite) Community im Diskussions Forum angewiesen ist. Ein/zwei meiner Fragen habe ich dann nachher auch im FAQ als Antwort gefunden, doch da die FAQ nicht strukturiert sind ist es nicht sehr offensichtlich dort zu suchen. Ich finde Tutorials sind der beste Weg um neuen Usern oder solche die ihre config yaml anpassen müssen Hilfestellung zur Verfügung zu stellen. Man müsste halt dort den Hinweis vermerken, dass sicth nicht alle Tutorials sich auf den aktuellen Stand von evcc beziehen und auch die Version angeben mit der das Tutorial erstellt worden ist. Ich habe für mich schon sehr rudimentäre Tutorials erstellt eines für OCPP und eines für HA mit MQTT.